rs/registry/fetch_large_record_test_canister/BUILD.bazel

```diff
@@ -0,0 +1,48 @@
+load("@rules_rust//rust:defs.bzl", "rust_library")
+load("//bazel:canisters.bzl", "rust_canister")
+
+package(default_visibility = [
+    "//rs/nervous_system/agent:__pkg__",
+    "//rs/registry/admin:__pkg__",
+])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/crypto/sha2",
+    "//rs/nervous_system/canisters",
+    "//rs/registry/transport",
+    "//rs/types/types",
+    "@crate_index//:candid",
+    "@crate_index//:ic-cdk",
+    "@crate_index//:serde",
+]
+
+MACRO_DEPENDENCIES = [
+    # Keep sorted.
+]
+
+LIBRARY_SRCS = glob(
+    ["src/**/*.rs"],
+    exclude = [
+        "src/main.rs",
+        "src/**/*tests.rs",
+    ],
+)
+
+rust_library(
+    name = "fetch_large_record_test_canister",
```

Comment:
Should be specified as a test-only target, I would say

Reply:
We agreed to move the implementation of agent's Request closer to the test, so that this PR only makes test changes. Then this crate can also be marked as test-only.